apps/ledger-live-desktop/src/renderer/components/Chart/index.tsx

```diff
@@ -36,10 +36,10 @@ import React, { useRef, useLayoutEffect, useState, useMemo } from "react";
 import ChartJs, { ChartColor, ChartData, ChartOptions, ChartTooltipModel } from "chart.js";
 import styled from "styled-components";
 import Color from "color";
-import moment from "moment";
 import useTheme from "~/renderer/hooks/useTheme";
 import Tooltip from "./Tooltip";
 import { Data, Item } from "./types";
+import { startOfHour, startOfDay } from "@ledgerhq/live-countervalues/portfolio";
```

Comment:
good point, i've created another task ( https://ledgerhq.atlassian.net/browse/LIVE-11877 ) because i think it's not the scope of `moment` removal.

the Chart displays the portfolio, portfolio that has moved to live-countervalues library due to its proximity to countervalues calculations, so it's not too far fetched that Chart needs the portfolio implementation.

That said, we could imagine our own dates utility library if we want to follow the path of modularisation.